desktop-app/app/utils/navigatorUtils.js

```diff
@@ -13,7 +15,10 @@ export function saveLastOpenedAddress(url) {
 }
 
 export function getHomepage() {
-  return settings.get(HOME_PAGE) || 'https://www.google.com/';
+  return userPreferenceSettings.getDefaultStartPage() ===
+    START_MANAGEMENT_OPTIONS.HOME
+    ? settings.get(HOME_PAGE) || 'https://www.google.com/'
+    : 'about:blank';
 }
```

Comment:
I think this function is used in more than one place, for example, what exactly happens in this scenario:
1- Open the app
2- Go to https://responsively.app
3- Set https://responsively.app as the home page 
4- Set the blank page as the startup page, 
5- Close the app
6- Open the app (the blank page should be opened)
7- Go to any website
8- Click on Go to Home Page button
9- **Does (should) it go to the black page or to responsively website ? 🤔**

Reply:
I thought that this was used only on startup. So I think you may be right that it should go to Home that was set as home and not the blank. That is at least what the expected result would be if I was looking at it. I will have to take a look at home this is used in the startup to use it there. Or do you think that this is an acceptable solution?

Reply:
That's what I think. The behavior after step 8 should be to go to the configured home page (https://responsively.app), not the startup page (_blank)